podcasts/Onboarding/Plus/PlusButtonStyles.swift

```diff
@@ -166,13 +166,13 @@ struct PlusFreeTrialLabel: View {
     }
 
     var body: some View {
-        Text(L10n.freeTrialDurationFreeTrial(text.localizedUppercase))
+        Text(text.localizedUppercase)
             .font(size: 12, style: .caption, weight: .semibold, maxSizeCategory: .extraExtraLarge)
             .multilineTextAlignment(.center)
             .padding([.top, .bottom], 4)
             .padding([.leading, .trailing], 13)
             .background(
-                color.cornerRadius(4)
+                color.cornerRadius(800)
```

Comment:
We can use the clipShape to achieve the same effect:

```suggestion
                color.clipShape(.capsule)
```